src/components/Company/Offers/Actions.js

```diff
@@ -0,0 +1,208 @@
+import React, { useCallback, useState } from "react";
+import PropTypes from "prop-types";
+import {
+    IconButton,
+    makeStyles,
+    TableCell,
+    Typography,
+} from "@material-ui/core";
+import { RowPropTypes } from "../../../utils/Table/PropTypes";
+import { getFieldValue } from "../../../utils/Table/utils";
+import { Check, Clear, ExpandMore, ExpandLess } from "@material-ui/icons";
+import { addSnackbar } from "../../../actions/notificationActions";
+import { connect } from "react-redux";
+// import ConfirmRejectDialog from "./ConfirmRejectDialog";
+
+const useStyles = makeStyles(() => ({
+    tableRowActions: {
+        // minWidth: "150px",
+    },
+}));
+
+/* TO BE DONE */
+
+const approveApplication = () => 0;
+
+const rejectApplication = () => 0;
```

Comment:
What's the purpose of these functions? What application are you referring to?

Reply:
Nothing. They shouldn't be there :upside_down_face:  I'm going to simplify the Actions.js file since it's not even being used right now. Could be in the future if we want the offers to open for more details, but for now I think having the Link to the Edit Offer fits better